### PR TITLE
Add a terminating newline at the end of generated endorsement statement

### PR DESCRIPTION
**Pull request by rbehjati**

### CODE REVIEW

File: cmd/endorser/main.go [hunk 1]
```diff
@@ -108,6 +108,9 @@ func main() {
 		log.Fatalf("Failed marshalling the endorsement: %v", err)
 	}
 
+	// Add a newline at the end of the file.
+	newline := []byte("\n")[0]
```
Comment (tiziano88) [inline cmd/endorser/main.go:112]: maybe

```suggestion
	newline := '\n'
```

Comment (andrisaar) [inline cmd/endorser/main.go:112]: Could you do something like `var newline byte = '\n'`? Going through a rune (and a slice) feels inelegant.

**Review by tiziano88 — APPROVED** [review]

**Review by andrisaar — APPROVED** [review]

File: cmd/endorser/main.go (hunk 1 above)
Comment (rbehjati) [reply cmd/endorser/main.go:112]: I tried, but it does not work. Here is the error: `cannot use newline (variable of type rune) as byte value in argument to append`. And I just learnt that `rune` is a 32 bit type. Not sure how to convert it to byte. Changed it to `newline := byte('\n')`.

Comment (rbehjati) [reply cmd/endorser/main.go:112]: Changed it to `newline := byte('\n')`.

**Pull request merged**